### ISSUE TITLE
About customizing clients in other languages

### ISSUE THREAD

**Issue by medz** [prisma/prisma-engines#2348]
Can I add a custom client document to the document?
For example, how to use the query engine, query parameters and the like.

**Comment by dpetrick** [comment]
Can you be a bit more specific? You mean the readme?

**Comment by medz** [comment]
> Can I add a custom client document to the document? For example, how to use the query engine, query parameters and the like.

@dpetrick No, I think it is possible to add documentation on how to customize the client using the engine in the Prisma documentation, such as how to use it, download the engine, and how to sideload examples.

**Comment by jkomyno** [comment]
Closing this for inactivity.
